sources/packages/web/src/views/aest/student/applicationDetails/NoticeOfAssessment.vue

```diff
@@ -12,7 +12,7 @@
     </template>
     <notice-of-assessment-form-view
       :assessment-id="assessmentId"
-      :can-reissue-m-s-f-a-a="true"
+      :can-reissue-m-s-f-a-a="hasStudentReissueMSFAARole"
```

Comment:
I believe that usually, the vue is disabling the component instead of hiding it. For this case to keep it simple I would no mind having it removed.